targets/ar71xx-generic/profiles.mk

```diff
@@ -169,3 +169,10 @@ $(eval $(call GluonProfileFactorySuffix,WNDR3700,.img))
 $(eval $(call GluonModel,WNDR3700,wndr3700-squashfs,netgear-wndr3700))
 $(eval $(call GluonModel,WNDR3700,wndr3700v2-squashfs,netgear-wndr3700v2))
 $(eval $(call GluonModel,WNDR3700,wndr3800-squashfs,netgear-wndr3800))
++
++## Allnet
++
++# ALL0315N
++$(eval $(call GluonProfile,ALL0315N,uboot-envtools,rssileds))
```

Comment:
Okay. By the way, there's no need to open a new pull request for fixups, you can just force-push over the branch the pull request was created from to update the request.